docs/build/eps.md

```diff
@@ -142,58 +142,70 @@ Dockerfile instructions are available [here](https://github.com/microsoft/onnxru
 ### Build Instructions
 {: .no_toc }
 
-These instructions are for JetPack SDK 4.6.1.
+These instructions are for the latest [JetPack SDK 5.1.2](https://developer.nvidia.com/embedded/jetpack-sdk-512).
 
 1. Clone the ONNX Runtime repo on the Jetson host
 
-    ```bash
-    git clone --recursive https://github.com/microsoft/onnxruntime
-    ```
+   ```bash
+   git clone --recursive https://github.com/microsoft/onnxruntime
+   ```
 
 2. Specify the CUDA compiler, or add its location to the PATH.
 
-   Cmake can't automatically find the correct nvcc if it's not in the PATH.
+   1. Starting with **CUDA 11.8**, Jetson users on **JetPack 5.0+** can upgrade to the latest CUDA release without updating the JetPack version or Jetson Linux BSP (Board Support Package). CUDA version 11.8 with JetPack 5.1.2 has been tested on Jetson when building ONNX Runtime 1.16. 
 
-    ```bash
-    export CUDACXX="/usr/local/cuda/bin/nvcc"
+      1. Check [this official blog](https://developer.nvidia.com/blog/simplifying-cuda-upgrades-for-nvidia-jetson-users/) for CUDA 11.8 upgrade instruction.
 
-    ```
+      2. CUDA 12.x might only be available to Jetson Orin and newer series. 
```

Comment:
remove the "might" language since it's a definite. 
we can say something like CUDA 12.x is supported on CUDA arch 8.7 and greater (Jetson Orin family)